acknowledgements.md

```diff
@@ -40,17 +40,17 @@ Really useful tool for benchmarking .NET code! Available for use under MIT Licen
 
 Even though they are no longer directly used for NSubstitute, the following projects were really helpful in building previous NSubstitute versions.
 
-## Microsoft Reactive Extensions for .NET (Rx) [http://msdn.microsoft.com/en-us/devlabs/ee794896]
-Used to provide .NET 3.5 with some of the neat concurrency helper classes that ship with out of the box with .NET 4.0. Distributed under a EULA [http://msdn.microsoft.com/en-us/devlabs/ff394099]. No longer required since we stopped supporting .NET 3.5.
+## Microsoft Reactive Extensions for .NET (Rx) [https://msdn.microsoft.com/en-us/devlabs/ee794896]
+Used to provide .NET 3.5 with some of the neat concurrency helper classes that ship with out of the box with .NET 4.0. Distributed under a EULA [https://msdn.microsoft.com/en-us/devlabs/ff394099]. No longer required since we stopped supporting .NET 3.5.
 
-## Microsoft Ilmerge [http://research.microsoft.com/en-us/people/mbarnett/ilmerge.aspx]
+## Microsoft Ilmerge [https://research.microsoft.com/en-us/people/mbarnett/ilmerge.aspx]
 Used for combining assemblies so NSubstitute can be distributed as a single DLL. Available for use under a EULA as described on the ilmerge site. No longer used now NuGet has taken over the .NET world.
 
-## 7-Zip [http://www.7-zip.org/]
-7-zip was used to zip up NSubstitute distributions as part of the automated build process. Distributed under a mixed GNU LGPL / unRAR licence [http://www.7-zip.org/license.txt]. No longer required as we just ship NuGet packages now.
+## 7-Zip [https://www.7-zip.org/]
+7-zip was used to zip up NSubstitute distributions as part of the automated build process. Distributed under a mixed GNU LGPL / unRAR licence [https://www.7-zip.org/license.txt]. No longer required as we just ship NuGet packages now.
 
-## NuPack [http://nupack.codeplex.com/]
-Used for packaging NSubstitute for distribution as a nu package. Distributed under the Apache License, Version 2.0 [http://www.apache.org/licenses/LICENSE-2.0.html]. Now longer required now NuGet has taken over the .NET world.
+## NuPack [https://nupack.codeplex.com/]
```

Comment:
I think we can drop this section now. I can't find a site for the old nupack project.

Reply:
yes, fine to me :)

Reply:
dropped